### DISCUSSION TITLE
Merge cells in table

### DISCUSSION THREAD

**Discussion by hyeokhyen** [greenelab/lab-website-template#291] [General] [answered]
### Checks

- [X] I have searched **[the docs](https://greene-lab.gitbook.io/lab-website-template-docs)**, [existing issues](https://github.com/greenelab/lab-website-template/issues), and [existing discussions](https://github.com/greenelab/lab-website-template/discussions) for answers first.

### Link to your website repo

https://github.com/KwonVitalLab/KwonVitalLab.github.io

### Version of Lab Website Template you are using

1.1.0

### Description

Could we merge cells in table? 
For example, can I have center-aligned "Spring Break" in the table?

|\# | Date | Theme | Topic | Reference (Chapter)  | Assignment |
|:- |:---- |:--- |:------------- |:----------- |:---------- |
| Spring Break| |
| | X  | X | | |

**Comment by vincerubinetti** [comment, marked as answer]
You can do this either by writing your table manually in HTML and using the [`colspan`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td#colspan) attribute, or using the [Jekyll Spaceship plugin that comes installed with later versions of the template](https://greene-lab.gitbook.io/lab-website-template-docs/advanced/jekyll-plugins) using [this syntax](https://github.com/jeffreytse/jekyll-spaceship?tab=readme-ov-file#rowspan-and-colspan).

![image](https://github.com/user-attachments/assets/73ff6b80-463e-446c-8b38-a1d24f17b26c)

**Reply by hyeokhyen** [reply]
Thank you for your reply. I am fairly new to this. Could you tell me how to install the later version of Jekyll, please?

**Reply by vincerubinetti** [reply]
> Thank you for your reply. I am fairly new to this. Could you tell me how to install the later version of Jekyll, please?

What do you mean by this? Are you referring to my comment about the later versions of the template? Because your version of the template has Jekyll Spaceship installed and enabled already. You should be able to just write that syntax and it should work.

**Reply by hyeokhyen** [reply]
Hmm... It didn't work for me. I tried coloring from the syntax page like "{: .cls style="background: orange" } " and merging columns to center the text across columns. But it didn't work. For example, I tried:

```md
|\# | Date | Theme | Topic | Reference (Chapter)  | Assignment |
|:- |:---- |:--- |:------------- |:----------- |:---------- |
| Spring Break {: .cls style="background: orange" } | |
| X  |
```

The bottom row X aligns with the 1st column.

**Reply by vincerubinetti** [reply]
`| Spring Break {: .cls style="background: orange" } | |`

You have a space between the two pipes. The Jekyll Spaceship readme shows `||` with no space.
